D1673/P1673.md

```diff
@@ -7208,17 +7208,6 @@ floating-point types or complex versions thereof, and if `T` has
 higher precision than `InVec1::value_type` or `InVec2::value_type`, then
 intermediate terms in the sum use `T`'s precision or greater.
 
-<i>[Note:</i> Like `reduce`, `dot` applies binary `operator+` in an
```

Comment:
Note to self: I commented this out already in PR https://github.com/ORNL/cpp-proposals-pub/pull/354 .